config/nav.yml

```
@@ -31,6 +34,9 @@ nav:
           - Sources, Brokers, and Triggers: getting-started/first-broker.md
           - Using a Knative Service as a source: getting-started/first-source.md
           - Using Triggers and sinks: getting-started/first-trigger.md
+        - More with Knative Functions:
+          - Next Steps: getting-started/function-next-steps.md
+          - Function Triggers and Event Sources: getting-started/function-triggers.md
```

```suggestion
         - Function Triggers and Event Sources: getting-started/function-triggers.md
          - Next Steps: getting-started/function-next-steps.md
```
I would put "Next Steps" last as there was nothing useful to run in my termal just docs and something about local builds that I feel too much work at this stage.